lib/Importer/ONNXModelLoader.cpp

```diff
@@ -486,6 +486,11 @@ llvm::Error ONNXModelLoader::loadConv(const ONNX_NAMESPACE::NodeProto &op,
     ASSIGN_VALUE_OR_RETURN_ERR(group, loadInt(dict.at("group")));
   }
 
+  unsigned_t dilation = 1;
+  if (dict.count("dilation")) {
+    ASSIGN_VALUE_OR_RETURN_ERR(dilation, loadInt(dict.at("dilation")));
```

Comment:
This doesn't look like it's an onnx conv attribute https://github.com/onnx/onnx/blob/master/docs/Operators.md#conv. onnx uses per-axis dilation

Reply:
Fair point. I made this change primarily to address dilation in a network coming from Caffe2 domain. I will modify the code to read dilation as a vector instead of scalar. I'm curious if anybody has encountered a case of dilation being two different values for different axis (x and y).

Reply:
You probably don't have to change dilation to vector everywhere, you could just read it as a vector in onnx model loader and return an Error if the dimensions are not the same. This may be exactly what you're saying, just making sure.

Reply:
Yeah that's what I plan to do.